src/components/graphics/Icon/Icon.tsx

```diff
@@ -21,6 +21,16 @@ export type Decoration = (
   | { type: 'background-circle' }
 );
 
+type EventProps = {
+  event: 'warning',
+};
+const Event = (props: EventProps) => {
+  const icon: IconName = props.event === 'warning' ? 'warning-filled' : props.event;
```

Comment:
Let's make it so that this only works for the valid types of the `event` prop. Currently it has `event: 'warning'`, so if `event` is not `warning` then this component should not render.